src/index.html

```diff
@@ -7,6 +7,365 @@
+
+          <div class="product-card__reviews">
+            <ul class="stars stars--4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+
+            <p class="product-card__reviews-text">Reviews: 5</p>
+          </div>
+
+          <div class="product-card__price">
+            <p class="product-card__price-label">Price:</p>
+
+            <p class="product-card__price-value">$2,199</p>
+          </div>
+
+          <a
+          href="#buy"
+          class="product-card__link"
+          data-qa="card-hover">
+          buy
+          </a>
+
+        </article>
+        <article class="product-card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27inch Retina 5K"
+            class="product-card__picture"
+          >
+
+          <h2 class="product-card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="product-card__code">Product code: 195434</p>
+
+          <div class="product-card__reviews">
+            <ul class="stars stars--4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+
+            <p class="product-card__reviews-text">Reviews: 5</p>
+          </div>
+
+          <div class="product-card__price">
+            <p class="product-card__price-label">Price:</p>
+
+            <p class="product-card__price-value">$2,199</p>
+          </div>
+
+          <a
+          href="#buy"
+          class="product-card__link"
+          data-qa="hover">
+          buy
+          </a>
+
+        </article>
+        <article class="product-card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27inch Retina 5K"
+            class="product-card__picture"
+          >
+
+          <h2 class="product-card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="product-card__code">Product code: 195434</p>
+
+          <div class="product-card__reviews">
+            <ul class="stars stars--4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+
+            <p class="product-card__reviews-text">Reviews: 5</p>
+          </div>
+
+          <div class="product-card__price">
+            <p class="product-card__price-label">Price:</p>
+
+            <p class="product-card__price-value">$2,199</p>
+          </div>
+
+          <a
```

Comment:
The same in this place and other same places also

Reply:
Have removed empty lines within each product-card